sdcm/cluster.py

```diff
@@ -1992,6 +1992,22 @@ def download_scylla_manager_repo(self, scylla_repo):
         if not self.is_rhel_like():
             self.remoter.run(cmd="sudo apt-get update", ignore_status=True)
 
+    @retrying(n=30, sleep_time=15, allowed_exceptions=UnexpectedExit)
+    def install_package(self, package_mgr, package_name):
+        wait.wait_for(func=self.is_apt_lock_free, step=30, timeout=60, text='Checking if package manager is free')
```

Comment:
looks like this calls apt-lock check for Centos/RH as well.
since no need calling this wait for non ubuntu/deb, it is better to add the 'if' condition here and not inside is_apt_lock_free.

Reply:
agree, it is much logical to use the `if self.distro.is_ubuntu` here, and we could possibly add other "waits" for other package manager, in case there will be others...
fixed.